ziti/contexts.go

```diff
@@ -4,103 +4,39 @@
 package ziti
 
 import (
-	"github.com/michaelquigley/pfxlog"
 	"github.com/openziti/edge-api/rest_model"
 	edge_apis "github.com/openziti/sdk-golang/edge-apis"
 	"github.com/openziti/sdk-golang/ziti/edge"
 	"github.com/openziti/sdk-golang/ziti/edge/posture"
 	cmap "github.com/orcaman/concurrent-map/v2"
 	"github.com/pkg/errors"
 	"net/url"
-	"os"
-	"path/filepath"
-	"strings"
-	"sync"
+	"strconv"
 )
 
-var contexts = new(sync.Map)
+var idCount = 0
 
-func init() {
-	ids := os.Getenv("ZITI_IDENTITIES")
+func NewId() string {
```

Comment:
Public function without doc. It's pretty straightforward imo but maybe continue the good vibes of adding the godoc?

Reply:
added